### PR TITLE
Issue 2312: add python client multi-topics consumer support

### PR DESCRIPTION
**Pull request by jiazhai**
### Motivation

app python client support for multi-topics consumer 

### Modifications

add 2 methods in python client
add unit test

### Result

all ut pass

### CODE REVIEW

File: pulsar-client-cpp/python/pulsar_test.py [hunk 1]
```diff
@@ -650,6 +650,110 @@ def _v2_topics(self, url):
...
+        topic3 = 'persistent://sample/standalone/ns/my-python-topics-consumer-3'
+        topics = [topic1, topic2, topic3]
+
+        url1 = self.adminUrl + '/admin/persistent/sample/standalone/ns/my-python-topics-consumer-1/partitions'
+        url2 = self.adminUrl + '/admin/persistent/sample/standalone/ns/my-python-topics-consumer-2/partitions'
+        url3 = self.adminUrl + '/admin/persistent/sample/standalone/ns/my-python-topics-consumer-3/partitions'
+
+        doHttpPut(url1, '2')
+        doHttpPut(url2, '3')
+        doHttpPut(url3, '4')
```
Comment (merlimat) [inline pulsar-client-cpp/python/pulsar_test.py]: As I mentioned in #2448, I think the multi-topic consumer in C++ is currently not working when there are no partitions in the topic.

File: pulsar-client-cpp/python/pulsar/__init__.py [hunk 2]
```diff
@@ -503,6 +503,211 @@ def my_listener(consumer, message):
         self._consumers.append(c)
         return c
 
+    def subscribe_topics(self, topics, subscription_name,
```
[the remark this answers is not in the document]
Comment (sijie) [reply pulsar-client-cpp/python/pulsar/__init__.py]: @jiazhai can you open a new issue addressing this comment?

File: pulsar-client-cpp/python/pulsar_test.py (hunk 1 above)
Comment (sijie) [reply pulsar-client-cpp/python/pulsar_test.py]: any actions to address for python after #2448?

**Comment by srkukarni** [comment]
Once #2448 is in, I will add support for functions as well.

**Pull request merged**